ocs_ci/ocs/node.py

```diff
@@ -2470,3 +2471,90 @@ def check_for_zombie_process_on_node(node_name=None):
             log.error(f"Zombie process found on node: {node_obj.name}")
             log.error(f"pid and ppid details: {out}")
             raise exceptions.ZombieProcessFoundException
+
+
+@switch_to_orig_index_at_last
+def get_provider_internal_node_ips(
+    node_type=constants.WORKER_MACHINE, num_of_nodes=None
+):
+    """
+    Get the provider node ips according to the node type (e.g. worker, master) and the
+    number of requested node ips from that type
+
+    Args:
+        node_type (str): The node type (e.g. worker, master)
+        num_of_nodes (int): The number of node ips to be returned
+
+    Returns:
+        list: The node ips
+
+    """
+    log.info("Switching to the provider cluster to get the provider node ips")
+    config.switch_to_provider()
+    provider_nodes = get_nodes(node_type, num_of_nodes)
+    provider_node_ips = [get_node_internal_ip(n) for n in provider_nodes]
+    log.info(f"Provider node ips: {provider_node_ips}")
+    return provider_node_ips
+
+
+def consumer_verification_steps_after_provider_node_replacement():
+    """
+    Check the consumer verification steps after a provider node replacement
```

Comment:
Could you please add those steps to the docstring. It is now unclear what is checked in this function.

Reply:
Yes, I will add it.